cfpb/consumerfinance.gov
# Upgrade to ccdb5-ui@2.7.0 | Upgrade to esbuild@0.18.20

See https://github.com/cfpb/ccdb5-ui/releases/tag/v2.7.0 for changes.

Fixes https://[GHE]/CFGOV/platform/issues/4515

## Changes

- Upgrade to ccdb5-ui@2.7.0
- Upgrade to esbuild@0.18.20


## How to test this PR

1. `yarn build` should pass without warnings.
2. http://localhost:8000/data-research/consumer-complaints/search/ localhost should work as before.
